src/ngraph/frontend/onnx_import/utils/common.cpp

```diff
@@ -60,23 +61,8 @@ namespace ngraph
                                       std::int64_t axis_range_min,
                                       std::int64_t axis_range_max)
             {
-                // Accepted range of value for axis is [axis_range_min, axis_range_max].
-                NGRAPH_CHECK(((axis >= axis_range_min) && (axis <= axis_range_max)),
-                             node.get_description(),
-                             "Parameter axis ",
-                             axis,
-                             " out of the tensor rank [-",
-                             axis_range_min,
-                             ", ",
-                             axis_range_max,
-                             "].");
-
-                if (axis < 0)
-                {
-                    axis = axis + tensor_rank;
-                }
-
-                return static_cast<size_t>(axis);
+                return ngraph::normalize_axis(
+                    node.get_description(), axis, tensor_rank, axis_range_min, axis_range_max);
```

Comment:
We can do this in a separate story, but we should remove the `validate_axis` function altoghether, as it now just adds an unneeded extra layer.

Reply:
I agree with you, but for transparency let's do it in separate PR.